pkg/ytconfig/canondata/TestGetControllerAgentsConfig/test.canondata

```diff
@@ -4,6 +4,12 @@
         "enable_ipv6"=%false;
         retries=1000;
     };
+    "solomon_exporter"={
+        host="{POD_SHORT_HOSTNAME}";
+        "instance_tags"={
+            "k8s_pod_name"="{K8S_POD_NAME}";
```

Comment:
I'd say that "k8s_" is excessive here, maybe just "pod_name", or even "pod"? I'd prefer having the most often used labels to be as concise as possible. Also, we have "host", not "host_name"/"hostname".

Reply:
-> pod